daemon/daemon.go

```diff
@@ -1095,6 +1095,7 @@ func (h *patchConfig) Handle(params PatchConfigParams) middleware.Responder {
 
 	// Only recompile if configuration has changed.
 	if changes > 0 {
+		d.policy.BumpRevision() // force policy recalculation
```

Comment:
Do we really need policy recalculation if the daemon's configuration is changed?

Reply:
Changes in policy enforcement have on effect on how the endpoint specific bpf program is generated. I changed the code in endpoint/policy.go to only regenerate if the policy changes, and did not want complicate that by, e.g., revisioning the option changes.